### PR TITLE
Updating to work with current version of React Native

### PR DESCRIPTION
**Pull request by AndrewHenderson**
I'm receiving the following error despite having no reference to `ReactNative.Component`.

screen shot 2016-11-01 at 1 05 42 pm

I think it might be the cache as you mentioned in your README, but clearing the cache did not resolve.

I figured I'd submit a PR to at least begin a conversation.

Also, it seems Xcode and the latest React Native made changes to several files.

### CODE REVIEW

File: app/native/components/HelloWorld.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import React, {
-  Component,
+import React, { Component } from 'react';
+import {
```
Comment (pakmee) [inline app/native/components/HelloWorld.js:2]: Proptypes is not part of react native anymore, you can move it beside the component import.

Comment (AndrewHenderson) [reply app/native/components/HelloWorld.js:2]: Ahh, good catch!

Comment (pakmee) [reply app/native/components/HelloWorld.js:2]: Btw, i tried the changes you added and I got it to build minus the proptypes stuff. Would be good if this PR goes in.

**Comment by AndrewHenderson** [comment]
Perfect. I'll fix it today 👍

File: app/native/components/HelloWorld.js (hunk 1 above)
Comment (AndrewHenderson) [reply app/native/components/HelloWorld.js:2]: Done 👍

**Comment by AndrewHenderson** [comment]
@kauffecup Btw, I noticed [you use `react-redux/native` for `connect`](https://github.com/kauffecup/react-native-web-hello-world/pull/2/files#diff-7a15a7427328a17053d8fe6c39b46c14L2).

Is there a difference? I've been using the standard one in my React Native app.

**Comment by kauffecup** [comment]
@AndrewHenderson thanks so much for taking the lead on this! i believe at the time of writing my original blog post there were different bindings for connect for native vs web, with the intentions to abstract those differences it

**Comment by AndrewHenderson** [comment]
@kauffecup I fixed [that issue](https://github.com/kauffecup/react-native-web-hello-world/pull/2#issuecomment-261680016) as well now ([see here](https://github.com/kauffecup/react-native-web-hello-world/pull/2/commits/118a07abee6825f10cde3e7796323c2d66100141)). Is there anything else you need on this before merging?

**Comment by kauffecup** [comment]
@AndrewHenderson and @pakmee just want to thank you both so much for this and apologize for my late response times! looks great!

**Pull request merged**